CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.6 (2019-07-05)
+
+- Updated development dependencies to support `@skyux-sdk/builder@3.7.1`. [#52](https://github.com/blackbaud/skyux-modals/issues/52)
```

Comment:
The link is to an issue, not a PR. Do we mean to link to PR 51 at https://github.com/blackbaud/skyux-modals/pull/51? I thought so at first, but the builder version in that PR is updated to 3.7.0, not 3.7.1. (Although `@skyux/i18n` is updated to 3.7.1)

Reply:
@blackbaud-johnly My mistake, I updated the link to point to a pull request. I updated the version of builder in this branch to `3.7.1`, even though the pull request updated builder to `3.7.0`. I don't think it matters, so long as we are communicating we updated Builder to `^3`.